.drone.yml

```diff
@@ -0,0 +1,24 @@
+---
+clone:
+  git:
+    image: plugins/git:next
+    pull: true
+
+pipeline:
+  test:
+    image: golang:1.12-buster
+    commands:
+      - export GO11MODULE=on
+      - make test
+
+  publish:
+    image: plugins/ecr
+    secrets: [ecr_access_key, ecr_secret_key, ecr_registry, ecr_repo]
```

Comment:
I don't think `ecr_registry, ecr_repo` are actually used here.

Reply:
They are used. Below in the `registry` and `repo` attributes of the settings.

Reply:
gotcha, just `ecr_repo` then.